src/components/slider/slider.js

```diff
@@ -237,9 +336,9 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
       }
 
       var changeAmount;
-      if (ev.keyCode === $mdConstant.KEY_CODE.LEFT_ARROW) {
+      if (vertical ? ev.keyCode === $mdConstant.KEY_CODE.DOWN_ARROW : ev.keyCode === $mdConstant.KEY_CODE.LEFT_ARROW) {
```

Comment:
I'm curious if we should just always associated left/down with negative and right/up with positive; no matter what orientation it's in.

Thoughts?